### ISSUE TITLE
Do we have the ability to customize your xml file?

### ISSUE THREAD

**Issue by anar1501** [BerryWorksSoftware/edireader#23]
Hello. First of all, thank you for doing this. There is one more thing I want to add from you. I saw using your project to create an xml that handles data exchange between un edifact passenger and airlines. What I want is this. Do we have the possibility to change the XML tags that are formed, so that we can create special tags for ourselves? Please I want you to help me with this, because I'm working on a project right now and I need to convert the edifact file pnrgov to xml and then java model classes. If it is convenient for you, I can assign you what kind of xml file to create specifically. I can also pay you for it.

**Comment by canabrook** [comment]
Yes, there are some options for customizing the XML tags. Please contact me at ***@***.*** or ***@***.*** for more info.
Scott

**Comment by anar1501** [comment]
I do not see your email please type me message I need to solve this problem if you wish. mammadov_anar@outlook.com
